steal.lic

```diff
@@ -18,16 +18,20 @@ class Steal
 
     UserVars.stealing_timers ||= {}
 
-    @stealing_options = get_data('stealing').stealing_options.reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
+    @stealing_options = get_data('stealing').stealing_options
+	  .reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
+	  .reject { |target| !settings.hometown.include?(target['steal_town'])} #added for to specify towns
```

Comment:
We can move @hometown declarations above this to shorten up the line, and use `==` instead of `include?` which is somewhat awkward when we just want exact string comparison.